webapp/src/main/java/org/apache/atlas/web/rest/TypesREST.java

```diff
@@ -459,6 +462,58 @@ public AtlasTypesDef createAtlasTypeDefs(final AtlasTypesDef typesDef) throws At
         }
     }
 
+    private AtlasBaseException checkIndexing(AtlasTypesDef typesDef) throws AtlasBaseException {
+
+
+        if (CollectionUtils.isNotEmpty(typesDef.getEntityDefs())) {
+            for (AtlasBaseTypeDef typedef : typesDef.getEntityDefs()) {
+                if (!typedef.getIndexCreated()) {
+                    return new AtlasBaseException(AtlasErrorCode.BAD_REQUEST, "Typedef not created in ES...");
+                }
+            }
+        }
+
+        if (CollectionUtils.isNotEmpty(typesDef.getEnumDefs())) {
+            for (AtlasBaseTypeDef typedef : typesDef.getEnumDefs()) {
+                if (!typedef.getIndexCreated()) {
+                    return new AtlasBaseException(AtlasErrorCode.BAD_REQUEST, "Typedef not created in ES...");
+                }
+            }
+        }
+
+        if (CollectionUtils.isNotEmpty(typesDef.getStructDefs())) {
+            for (AtlasBaseTypeDef typedef : typesDef.getStructDefs()) {
+                if (!typedef.getIndexCreated()) {
+                    return new AtlasBaseException(AtlasErrorCode.BAD_REQUEST, "Typedef not created in ES...");
+                }
+            }
+        }
+        if (CollectionUtils.isNotEmpty(typesDef.getClassificationDefs())) {
+            for (AtlasBaseTypeDef typedef : typesDef.getClassificationDefs()) {
+                if (!typedef.getIndexCreated()) {
+                    return new AtlasBaseException(AtlasErrorCode.BAD_REQUEST, "Typedef not created in ES...");
+                }
+            }
+        }
+
+        if (CollectionUtils.isNotEmpty(typesDef.getRelationshipDefs())) {
+            for (AtlasBaseTypeDef typedef : typesDef.getRelationshipDefs()) {
+                if (!typedef.getIndexCreated()) {
+                    return new AtlasBaseException(AtlasErrorCode.BAD_REQUEST, "Typedef not created in ES...");
+                }
+            }
+        }
+
+        if (CollectionUtils.isNotEmpty(typesDef.getBusinessMetadataDefs())) {
+            for (AtlasBaseTypeDef typedef : typesDef.getBusinessMetadataDefs()) {
+                if (!typedef.getIndexCreated()) {
+                    return new AtlasBaseException(AtlasErrorCode.BAD_REQUEST, "Typedef not created in ES...");
+                }
+            }
+        }
+        return null;
```

Comment:
Donot return `null`. Instead make this method void, since we are doing boolean checks.